### ISSUE TITLE
[bitnami/external-dns] Add traefik-proxy rules to external-dns helm chart

### ISSUE THREAD

**Issue by trunet** [bitnami/charts#19211]
### Name and Version

bitnami/external-dns 6.24.3

### What is the problem this feature will solve?

To work with traefik, the ClusterRole needs access to its CRDs.

### What is the feature you are proposing to solve the problem?

Add the RBAC as external-dns did in their own helm chart, implemented in this PR: https://github.com/kubernetes-sigs/external-dns/pull/3325

### What alternatives have you considered?

Moving to external-dns own helm charts, but I found rfc2136 bitnami implementation better, and want to stick with it.

**Comment by carrodher** [comment]
Thank you for opening this issue and submitting the associated Pull Request. Our team will review and provide feedback. Once the PR is merged, the issue will automatically close.

Your contribution is greatly appreciated!

**Comment by trunet** [comment]
This will be fixed by https://github.com/bitnami/charts/pull/19168. I closed my PR because is a duplicate of an earlier one.
